tracks/ruby/exercises/grains/mentoring.md

```diff
@@ -19,26 +19,36 @@ module Grains
   def self.total
     square(SQUARES.max) * 2 - 1
   end
+
 end
 
 class BoardPositionError < ArgumentError
+
```

Comment:
Personally, I'm not a fan of these additional empty lines (it won't block approval)